README.md

```diff
@@ -2,19 +2,39 @@
 
 > IPA narrow transcription of English words in New York City accent
 
-## Demo Website
+## Online Demo Website
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/6cc76b39-d72c-4d05-9959-0d57d6b8b6e4/deploy-status)](https://app.netlify.com/sites/ipa-jfk/deploys)
 
 You can lookup words on our hassle-free **[Demo Website](https://jfk.b1f6c1c4.info/)**!
 
-## Install
+## Prerequisites
+
+- Node.js 12.x+
+- Yarn (recommended)
+- [Parcel](https://v2.parceljs.org/) (for demo)
```

Comment:
Inconsistent with `parcel-bundler: "^1.12.5"`

Reply:
Stated that it is not that easy to upgrade to `parcel` v2.

Reply:
> Stated that it is not that easy to upgrade to `parcel` v2.

See also [migration](https://v2.parceljs.org/getting-started/migration/) and [known issues](https://v2.parceljs.org/blog/rc0/).

Reply:
Then you should point to `v1` of `parcel`

Reply:
```suggestion
- [Parcel](https://parceljs.org/getting_started.html) (for demo)
```